google/cloud/bigtable/benchmarks/embedded_server_test.cc

```diff
@@ -18,7 +18,8 @@
 #include <gmock/gmock.h>
 #include <thread>
 
-using namespace bigtable::benchmarks;
+using namespace google::cloud::bigtable::benchmarks;
```

Comment:
Can this be dropped, and then you can leave the code unchanged (`bigtable::benchmarks` still works, no?)